kubernetes/Memory.dhall

```diff
@@ -0,0 +1,58 @@
+let Memory =
```

Comment:
You can use Space for that, it is exactly the same and it is used for the same purpose (memory space, disk space, etc...)